src/main/webapp/index.html

```diff
@@ -30,18 +30,20 @@
                         <ul id="navbar">
                             <li class="current"><strong>Home</strong></li>
                             <li><a href="features.html">Features</a></li>
-                            <li><a href="about.html">About Us</a></li>
                             <li><a href="contact.html">Contact</a></li>
+                            <li><a href="about.html">About Us</a></li>
                             <li><a href="terms.html">Terms of Use</a></li>
                         </ul>
                     </div>
                    <div id="loginHolder">
-                            <form action="/login" style="float: left;" name="studentLogin">
-                            <input type="submit" name="student" class="button" id="btnStudentLogin" value="Student Login">
-                        </form>
-                        <form action="/login" name="instructorLogin" style="float: left;">
-                            <input type="submit" name="instructor" class="button" id="btnInstructorLogin" value=" Instructor Login">
-                        </form>
+                          <form action="/login" name="instructorLogin" style="float: left;">
+                                <input type="submit" name="instructor" class="button" id="btnInstructorLogin" value=" Instructor Login"/>
+                          </form>
+                          
+                          <form action="/login" style="float: left;" name="studentLogin">
+                            <input type="submit" name="student" class="button" id="btnStudentLogin" value="Student Login"/>
+                         </form>
+                        
                    </div>
```

Comment:
You should make the changes for the issue in a new branch, instead of the 'tinker' branch